apps/src/code-studio/components/AssetRow.jsx

```diff
@@ -51,11 +53,24 @@ export default class AssetRow extends React.Component {
     });
   };
 
+  chooseAsset = () => {
+    firehoseClient.putRecord(
+      {
+        study: 'sound-dialog',
+        study_group: experiments.isEnabled(experiments.AUDIO_LIBRARY_DEFAULT) ? 'library-tab' : 'files-tab',
+        event: 'choose-uploaded-sound',
+        data_json: this.props.name
+      },
+      {includeUserId: true}
+    );
+    this.props.onChoose();
+  };
```

Comment:
It doesn't look like this code is specific to audio - do we intend to get these events any time someone picks any asset, or only for sounds?